yt_dlp/extractor/ant1newsgr.py

```diff
@@ -0,0 +1,184 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+from urllib.parse import (
+    parse_qs,
+    urlparse,
+    urlunparse,
+)
```

Comment:
```suggestion
import urllib.parse
```
to avoid any confusion. We have similarly named functions in utils/compat